### ISSUE TITLE
Got many more flares than tutorial

### ISSUE THREAD

**Issue by marioraulhq** [afeinstein20/stella#12]
Hi!

While working through the tutorial, I found that ff.flare_table is giving me many more flares than what the tutorial suggests. At a threshold of 0.5, the tutorial says that there are 16 such flares while I'm getting 31. I got the same result using both the pip and GitHub version. At first I thought I might have made a mistake somewhere, so I decided to run the exact same thing that's shown in [https://adina.feinste.in/stella/getting_started/other_features.html](url)- I still got the same thing. 

Do you have any suggestions about what I could do to work through this?

Thanks,
Mario H.

**Comment by afeinstein20** [comment]
Hi Mario,

Sorry for such a delayed response to this issue. Could you please send me what version of tensorflow, numpy, and scipy you are using? This may help me diagnose what is going wrong.

Thanks!!

**Comment by marioraulhq** [comment]
Hi Adina,

No worries, and I apologize for my equally long reply too. With all the new changes and versions that have came out since I originally made my post, I don't think this ticket is too much of an issue anymore. Besides, I think I got everything to work as intended. Thanks for the reply!

**Comment by aftab234** [comment]
@afeinstein20 Sorry for posting this in a ticket that was closed so long ago. I get 58 flares when I use one of the pre-trained models and follow the "Other Fun Features" tutorial. Many duplicates pop up in the table. Probably a problem with newer versions of the dependencies?
![image](https://github.com/afeinstein20/stella/assets/72819040/1177492e-0cc8-4500-80a7-9461bb4c8bbb)


Also, I noticed the tutorial has the equivalent duration in minutes while the most recent version of stella has it in seconds.
